config/routes.rb

```diff
@@ -218,6 +218,9 @@
       get 'resolve'
       put 'resolve'
     end
+    collection do
```

Comment:
What does `collection` do versus `member`? This is a bit of Rails I'm not familiar with.

Reply:
`member` always create routes that require `id` to perform some action. It usually used for a route that actions on a unique instance.

`collection` is used for performing operations over a group or where there is no ID required. Such as in this case where we need to create Alert and there is no ID yet.

In collection our route becomes `/notify_instructors`, and if `member`  route would have become `/notify_instructors/:id`